service/controller/clusterapi/v28/key/machine_deployment.go

```diff
@@ -60,3 +106,41 @@ func WorkerScalingMax(cr v1alpha1.MachineDeployment) int {
 func WorkerScalingMin(cr v1alpha1.MachineDeployment) int {
 	return machineDeploymentProviderSpec(cr).NodePool.Scaling.Min
 }
+
+// calculateSubnetMask calculates new subnet mask to accommodate n subnets.
+func calculateSubnetMask(networkMask net.IPMask, n uint) (net.IPMask, error) {
```

Comment:
Can't the `network` package provide this functionality? Especially when we move stuff around later this should not be specific to the `key` package.

Reply:
Can. I was already thinking that, but then kind of resolved to this "refactor on third repetition" "rule" :slightly_smiling_face: I'll move this.